core/App/utils/helpers.ts

```diff
@@ -28,9 +28,28 @@ export function hashCode(s: string): number {
   return s.split('').reduce((hash, char) => char.charCodeAt(0) + ((hash << 5) - hash), 0)
 }
 
-export function hashToRGBA(i: number) {
-  const colour = (i & 0x00ffffff).toString(16).toUpperCase()
-  return '#' + '00000'.substring(0, 6 - colour.length) + colour
+/**
+ * seeded random number generator between 0-1
+ * Adapted from https://stackoverflow.com/questions/521295/seeding-the-random-number-generator-in-javascript
+ */
+function mulberry32(a: number) {
+  let t = (a += 0x6d2b79f5)
```

Comment:
Quite a few [magic numbers](https://en.wikipedia.org/wiki/Magic_number_(programming)) here. What's so special about 15, 1, 7, 61, 14? Some times adding them as descriptive variable names will help the next person who needs to maintain your code:

`const theAnswerToEverything = 42`